lbuild/environment.py

```diff
@@ -203,6 +204,8 @@ def template(self, src, dest=None, substitutions=None, filters=None):
             else:
                 dest = src
 
+        src = self.repopath(src)
```

Comment:
If this produces a path outside the repository (i.e. starting with "../") then the template load would fail again. I think it is useful to check that here and throw an exception with a descriptive error message.